src/ngraph/graph_util.hpp

```diff
@@ -75,6 +75,8 @@ namespace ngraph
 
     void replace_node(std::shared_ptr<Node> target, std::shared_ptr<Node> replacement);
 
+    NodeVector find_common_args(std::shared_ptr<Node> target, std::shared_ptr<Node> replacement);
```

Comment:
Minor thing: these parameter names should be changed, unless I'm misunderstanding the purpose of the function (in which case the function should probably be renamed).

Reply:
I think you are not misunderstanding. I agree the parameter names should be more generic.